internal/entry/types.go

```diff
@@ -0,0 +1,21 @@
+package entry
```

Comment:
why have you renamed Labels to Entry?

Reply:
last name for thi type was container I would like something who work in both case but we can change entry by label if ou want